src/services/GroupService.js

```diff
@@ -192,10 +192,10 @@ async function createGroup(currentUser, data) {
     await helper.postBusEvent(config.KAFKA_GROUP_CREATE_TOPIC, group)
     await tx.commit()
 
-    // set the cache
-    const cache = await helper.getCacheInstance()
-    cache.set(group.id, group)
-    cache.set(`${group.id}-members`, [])
+    // do not set the cache as while creating the groups is not having the `oldId`
```

Comment:
should we remove those lines instead of commenting them out?

Reply:
I have kept them as we need them once we introduce DAL, where we would be getting the `oldId` immediately and then we can put the group in the cache.